model/src/main/scala/ch.epfl.scala.index.model/SemanticVersion.scala

```diff
@@ -68,20 +68,40 @@ case class SemanticVersion(
 }
 
 object SemanticVersion extends Parsers {
-  private implicit val preReleaseOrdering: Ordering[PreRelease] = Ordering.by {
-    case ReleaseCandidate(rc) => (Some(rc), None, None)
-    case Milestone(m) => (None, Some(m), None)
-    case OtherPreRelease(pr) => (None, None, Some(pr))
-  }
+  private implicit val preReleaseOrdering: Ordering[Option[PreRelease]] =
+    Ordering.by {
+      case None                       => (3, None, None)
+      case Some(ReleaseCandidate(rc)) => (2, Some(rc), None)
+      case Some(Milestone(m))         => (1, Some(m), None)
+      case Some(OtherPreRelease(pr))  => (0, None, Some(pr))
```

Comment:
Wouldn't the following be enough?
```suggestion
      case None                       => (3, None)
      case Some(ReleaseCandidate(rc)) => (2, Some(rc))
      case Some(Milestone(m))         => (1, Some(m))
      case Some(OtherPreRelease(pr))  => (0, Some(pr))
```

Reply:
It is deployed to http://index-dev.scala-lang.org

Reply:
No, its not enough because `pr` is a String whereas `rc` and `m` are Long

Reply:
Oh, OK.